examples/bri-3/src/bri/zkp/api/dtos/request/createProof.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsNotEmpty } from 'class-validator';
+
+// TODO: Revisit validations rules once we move into business logic implementation
+export class CreateProofDto {
+  @IsNotEmpty()
+  id: string;
```

Comment:
do we need id in create dto or we can generate uuid?

Reply:
lets us uuid, pls